client/src/game/ui/SelectionInfo.vue

```diff
@@ -69,6 +70,15 @@ function setValue(data: { solution: number; relativeMode: boolean }): void {
     }
 }
 
+function updateAura(aura: DeepReadonly<UiAura>, delta: Partial<Aura>, syncTo = true): void {
+    if (delta.value !== undefined && (isNaN(delta.value) || delta.value < 0)) delta.value = 0;
+    if (delta.dim !== undefined && (isNaN(delta.dim) || delta.dim < 0)) delta.dim = 0;
+    if (aura.temporary) {
+        auraSystem.add(aura.shape, { ...aura }, SERVER_SYNC);
+    }
```

Comment:
I actually don't think you need to worry about these things. With the change you made, we're no longer modifying the `.value` or `.dim` and the temporary auras don't show up in the selection info either.

Reply:
all right, I will remove that.
btw, what does "NO_SYNC" do actually? It seems to keep sync anyway.

Reply:
Yeah so while the mouse is still pressed, the changes are only applied locally. When the mouse is released, it's properly synced to the server and all other players.